src/Components/Login/login.css

```diff
@@ -1,13 +1,35 @@
+.title {
```

Comment:
pls geen position absolute.. Je kan beter even kijken naar hoe flex box werkt; 
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox
https://css-tricks.com/snippets/css/a-guide-to-flexbox/